charts/fleet/102.1.0+up0.7.0-rc.2/Chart.yaml

```
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/auto-install: fleet-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
+  catalog.cattle.io/namespace: cattle-fleet-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/provides-gvr: clusters.fleet.cattle.io/v1alpha1
+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
```

```suggestion
  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
```

Maybe a stupid question what is this annotation used for? And why the fleet-crd do not have the same?

---

Not sure, I will look into it.

---

It seems to be generated by one of the scripts in the charts repo.